### PR TITLE
Remove unused check for Stream length

### LINKED ISSUE #345: Dev branch Stream length not supported
image

### PR DESCRIPTION
**Pull request by taooceros**
fix #345 

The cancellation process is done by HttpClient natively by throwing TaskCancelledException, so no need to check the Stream length for that.

Seems that throwing exceptions makes the programs runs really slow in debug mode? Not quite sure for that.

### CODE REVIEW

**Comment by jjw24** [comment]
>Seems that throwing exceptions makes the programs runs really slow in debug mode? Not quite sure for that.

is this referring to throwing cancellation token exceptions?

**Comment by taooceros** [comment]
> > Seems that throwing exceptions makes the programs runs really slow in debug mode? Not quite sure for that.
> 
> is this referring to throwing cancellation token exceptions?

I believe, but not quite sure.

File: Plugins/Flow.Launcher.Plugin.WebSearch/SuggestionSources/Google.cs [hunk 1]
```diff
@@ -24,19 +24,16 @@ public override async Task<List<string>> Suggestions(string query, CancellationT
...
             }
             catch (TaskCanceledException)
             {
-                return null;
+                return new List<string>();
             }
             catch (HttpRequestException e)
             {
-                Log.Exception("|Google.Suggestions|Can't get suggestion from google", e);
+                // Log.Exception("|Google.Suggestions|Can't get suggestion from google", e);
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.WebSearch/SuggestionSources/Google.cs:36]: commented out?

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.WebSearch/SuggestionSources/Google.cs:36]: Oops. for debugging.😅

**Review by jjw24 — APPROVED** [review]

**Pull request merged**